### ISSUE TITLE
Keras in R

### ISSUE THREAD

**Issue by KafeelBasha** [rstudio/keras#371]
How to decide the value of 'filters' parameter in layer_conv_2d()?
What is the difference between padding="same" and padding="valid"?

If possible please provide an example when number of filters are 4.

**Comment by dfalbel** [comment]
Here is a great explanation of strides and padding: https://github.com/vdumoulin/conv_arithmetic#convolution-animations

Filters are the depth of the output (or the number of channels) of the convolutional network. This is a hyperparameter, to decide it's value you will need to use some hyperparameter search technique.

**Comment by KafeelBasha** [comment]
Thanks a lot.
